datafusion/core/Cargo.toml

```diff
@@ -85,7 +81,7 @@ lazy_static = { version = "^1.4.0" }
 log = "^0.4"
 num-traits = { version = "0.2", optional = true }
 num_cpus = "1.13.0"
-object_store = "0.5.4"
+object_store = { version = "0.5.4", features = ["aws"] }
```

Comment:
I'm concerned about introducing cloud storage features into datafusion/core. Perhaps we could push down options into object_store through a general-purpose interface and handle the s3/oss logic there.

Reply:
Thinks for this feedback.
I think the question here is whether we enable access to cloud storage by default for datafusion. If yes, it is the same whether to put the options in datafusion or in object_store. If not, I can make this as an optional feature for datafusion/core, and turn it on by default in the datafusion-cli.
Which do you think is better? happy to discuss this 😄